### DISCUSSION TITLE
BSC_project_IPFS2023 - Week9: Grain development v

### DISCUSSION THREAD

**Discussion by giscus** [Illustratien/BSC_project_IPFS2023#3] [General]
# BSC_project_IPFS2023 - Week9: Grain development v

https://illustratien.github.io/BSC_project_IPFS2023/posts/Week9/

**Comment by Illustratien** [comment]
https://box.hu-berlin.de/d/fc7db144021a4a35a086/?dl=1

**Comment by WasellaN** [comment]
https://box.hu-berlin.de/d/46b9e6f4fafd48929715/

**Comment by hanwenhsu** [comment]
https://box.hu-berlin.de/d/f22a20efa64149f88ed9/

**Comment by hanwenhsu** [comment]
https://github.com/hanwenhsu/Ear_development_BSC_project_hwh

**Comment by WasellaN** [comment]
https://github.com/WasellaN/Ear_Project_Nouri

**Reply by Illustratien** [reply]
Hey @WasellaN,
Your cut function is not right.
you can examine with 
```
ggplot(df,aes(type,spike))+
  geom_point()
```

**Comment by SaklineShawon-7** [comment]
https://github.com/SaklineShawon-7/Ear_development_BSC_project.git

**Comment by HU-Studienprojekt-2023Winter** [comment]
https://github.com/HU-Studienprojekt-2023Winter/Wheat_BSC_project
